core/common/src/serializers/InstantSerializers.kt

```diff
@@ -15,19 +17,18 @@ import kotlinx.serialization.encoding.*
  *
  * JSON example: `"2020-12-09T09:16:56.000124Z"`
  *
- * @see Instant.toString
- * @see Instant.parse
+ * @see DateTimeComponents.Formats.ISO_DATE_TIME_OFFSET
  */
 public object InstantIso8601Serializer : KSerializer<Instant> {
 
     override val descriptor: SerialDescriptor =
-        PrimitiveSerialDescriptor("kotlinx.datetime.Instant", PrimitiveKind.STRING)
+        PrimitiveSerialDescriptor("kotlinx.datetime.Instant ISO", PrimitiveKind.STRING)
```

Comment:
While there are no apparent restrictions on what can be used as a serial name, and no recommendations how to name multiple alternate serializers for the same type, this name may be used in error messages without quotes, so I think it would be better not to use spaces, so it can be read as one word.
Perhaps a slash can be used? E.g. `kotlinx.datetime.Instant/ISO` and `kotlinx.datetime.Instant/components`
Or have we already come to a decision about the current serial descriptor naming scheme?

Reply:
I do remember that we have explicitly (but arbitrarily!) chosen spaces, yes, but the point about error messages was not raised at the time. The naming scheme you propose looks good.